polaris-react/src/components/Frame/components/ContextualSaveBar/ContextualSaveBar.tsx

```diff
@@ -12,7 +12,6 @@ import type {ContextualSaveBarProps} from '../../../../utilities/frame';
 import {getWidth} from '../../../../utilities/get-width';
```

Comment:
[This PR](https://5d559397bae39100201eedc1-tnpnnmnlft.chromatic.com/?path=/story/all-components-contextualsavebar--default)
[Production](https://storybook.polaris.shopify.com/?path=/story/all-components-contextualsavebar--default&globals=polarisSummerEditions2023:true)